examples/customresources/models/mynavigation/mynavigation.go

```diff
@@ -18,19 +18,47 @@ import (
 var Model = resource.NewModel("acme", "demo", "mynavigation")
 
 func init() {
-	resource.RegisterService(navigation.API, Model, resource.Registration[navigation.Service, resource.NoNativeConfig]{
+	resource.RegisterService(navigation.API, Model, resource.Registration[navigation.Service, *Config]{
 		Constructor: newNav,
 	})
 }
 
+// Config is the navigation model's config.
+type Config struct {
+	Lat  *float64 `json:"lat,omitempty"` // omitempty for a pointer to a float64 defaults to nil in golang
+	Long *float64 `json:"long,omitempty"`
+
+	// a helper function to denote that we are not checking if any attributes exist or are set to anything in particular
```

Comment:
```suggestion
	// Embed TriviallyValidateConfig to make config validation a no-op. We will not check if any attributes exist or are set to anything in particular, and there will be no implicit dependencies.
```
Slight elaboration/correction. `TriviallyValidateConfig` is not really a helper function but a struct you can embed that already implements `Validate`. Same goes for `mysum.Config`.